packages/shared/src/graphql/sources.ts

```diff
@@ -13,6 +28,10 @@ export interface Source {
   handle: string;
   type: SourceType;
   permalink: string;
+  membersCount: number;
+  description?: string;
+  members?: Connection<SourceMember>;
+  currentMember?: SourceMember;
```

Comment:
The build will fail on token page. The issue is, some parts were using the source member's source property, then its description. We can move all the types from `/squads` file to `/sources` file altogether if want this to pass and remove those properties back to Squad type. WDYT?

Reply:
Wait, let me try to mix some things up.

Reply:
I don't think it is possible to fix it since literally the one file is referencing the other. Bundler will probably be able to resolve it but linter just forbids it. 

I think it is fine to refactor to move used types to common or something.

Reply:
Yep probably move them to a common is actually a neater solution down the line.
But yes let's keep types where they belong and not make shortcuts because lint/type issues for now.

Reply:
I have simply moved the Squad to the Source file since it basically extends Source. Build should be fixed now.